### DISCUSSION TITLE
Better diff highlights

### DISCUSSION THREAD

**Discussion by dudicoco** [catppuccin/nvim#692] [Q&A] [answered]
Hi,

The diff highlights for changed lines and changed characters within changed lines are very subtle:
![Screenshot 2024-04-05 at 21 04 36](https://github.com/catppuccin/nvim/assets/51051860/cae65164-7c0e-4457-86d0-e150c358f3ad)

I've tried overwriting the changed diff highlights [here](https://github.com/catppuccin/nvim/blob/aebe43db9cb26e1c70fc5b2fd4158169c405e720/lua/catppuccin/groups/syntax.lua#L65-L76) to green without success.

Where are these highlights set?

Thanks

**Comment by mrtnvgr** [comment]
You can check highlights under your cursor with `:Inspect`

**Reply by dudicoco** [reply]
@mrtnvgr this gives me the treesitter types but not the highlights:
```
Treesitter
  - @text.title.2.markdown links to rainbow2 markdown
  - @spell.markdown links to @spell markdown
```

**Reply by mrtnvgr** [reply]
You can override them just like any other highlights:

```vim
:hi! link @text.title.2.markdown Error
```

**Reply by dudicoco** [reply]
@mrtnvgr how would that work exactly? I am looking to override the diff highlights, not the `@text.title.2.markdown` highlight.

**Reply by mrtnvgr** [reply]
I just noted that these are normal groups that you can easily override and gave an example :)

What plugin do you use? 🤔

**Reply by dudicoco** [reply]
@mrtnvgr https://github.com/sindrets/diffview.nvim

**Reply by mrtnvgr** [reply, marked as answer]
Try changing `DiffAdd`, `DiffChange`, `DiffDelete`, `DiffText` groups
<br>

![20240406_21h09m01s_grim](https://github.com/catppuccin/nvim/assets/48406064/783a38a6-5b46-4472-903e-edadfbf79695)
`:hi! link DiffAdd Cursor`
![20240406_21h09m57s_grim](https://github.com/catppuccin/nvim/assets/48406064/c7ec2a93-b181-446e-9056-3a3dce9f2f14)

**Reply by dudicoco** [reply]
@mrtnvgr I've tried these before but with `fg`, used `bg` now and problem is solved. Thanks!
